src/client/views/components/todoItem.jsx

```diff
@@ -19,6 +19,14 @@ export default React.createClass({
         if (this.state.inEdit) {
             this.focusOnHeading();
         }
+
+        let el = this.getDOMNode();
+        let isVisible = (el.getBoundingClientRect().top >= 0) && (el.getBoundingClientRect().bottom <= window.innerHeight);
+        if (!isVisible) {
+            setTimeout(() => {
+                window.scrollTo(0,document.body.scrollHeight);
+            }, 100);
```

Comment:
The height takes 150ms to full animate, which is why I think it is. I tried changing the time and noticed the scroll was less... which leads me to think that's the cause.